### PR TITLE
Fixes for Consent page for Shield study 7 - Activity Stream

### PR DESCRIPTION
**Pull request by tspurway**
- these changes reflect changes in master document over here: https://docs.google.com/document/d/1CmUCh_96b4idf8X4yfXn-BT7P47TRSAevy4s4YdFapQ/edit#

### CODE REVIEW

**Comment by muffinresearch** [comment]
@tspurway hi when does shield study 7 need to be live? If it needs to make this week's release this will need to be cherry-picked onto the tag that was created yesterday. 

It's also worth noting the other commit for shield study 7 is already going out this week.

File: src/olympia/pages/templates/pages/shield_study_7.html [hunk 1]
```diff
@@ -40,31 +40,24 @@ <h3 id="all-shield-studies">All Shield Studies</h3>
...
-<li>An <em>Event</em> ping will be sent to Mozilla servers when you interact with the about:newtab page (Top Sites, Highlights, Search).</li>
-<li>A <em>Performance</em> ping will be sent to Mozilla servers occasionally to help track responsiveness and latency.</li>
+<li>We collect basic usage data regarding how you interact with the experimental new tab and timeline pages, such as the types of elements that you click and their position on the page. </li>
+<li>We also collect basic information about your browser profile that affect the study, such as the number of bookmarks and history items.</li>
+<li>To provide this new experience, we fetch metadata about websites you have visited, such as images and icons, using a service called <a href="http://embed.ly/">Embedly</a>. We send these requests to Embedly through Mozilla’s servers, so they will not be associated with you.</li>
 <li>As you would expect, we do not collect any information when you are in private browsing mode.</li>
 </ul>
 <p>You can learn more about the data collection for this study <a href="https://github.com/mozilla/activity-stream/blob/master/data_events.md">here</a></p>
```
Comment (muffinresearch) [inline src/olympia/pages/templates/pages/shield_study_7.html]: To improve the accessibility of this link could it be wrapped around more than just the word "here" to provide more context?

**Comment by tspurway** [comment]
@madalincm  It would be great if this patch could be uplifted/cherrypicked if possible.  We would like to get get the study started asap.

**Comment by muffinresearch** [comment]
@tspurway thanks - I'll get this added to the picks list.

**Pull request merged**